src/components/Checkbox/Checkbox.tsx

```diff
@@ -21,6 +22,15 @@ export const Checkbox = componentMapper<CheckboxProps>(SelectionControl, {
 
 		return color;
 	},
+	onPress: ({ onPress, onValueChange, value, checked }: any) => ()  => {
+		if (onPress) {
+			onPress();
+		}
+
+		if (onValueChange) {
+			onValueChange(value, checked);
```

Comment:
Is the new value sent forward correct? Or should it be `!checked`?

Reply:
It will pass the current value of the checkbox before alteration.
This is the behaviour followed by material ui 
React Native paper only have an onPress which does not provide you any parameters and it actually expects you to alter the current state manually.